docs/vacanza.py

```
@@ -0,0 +1,61 @@
+            holidays = country_holidays(
+                country_code,
+                years=years,
+                categories=cat
+            )
+
+            filename = f"{country_code}_{cat}.ics"
+            ICalExporter(holidays).save_ics(filename)
+
+            print(f"Generated: {filename}")
+
+    except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)